src/python_testing/TC_TIMESYNC_2_1.py

```diff
@@ -41,40 +41,47 @@ def pics_TC_TIMESYNC_2_1(self) -> list[str]:
 
     @async_test_body
     async def test_TC_TIMESYNC_2_1(self):
+        endpoint = 0
 
-        endpoint = self.user_params.get("endpoint", 0)
+        features = await self.read_single_attribute(dev_ctrl=self.default_controller, node_id=self.dut_node_id,
+                                                    endpoint=endpoint, attribute=Clusters.TimeSynchronization.Attributes.FeatureMap)
+
+        self.supports_time_zone = bool(features & Clusters.TimeSynchronization.Bitmaps.Feature.kTimeZone)
+        self.supports_ntpc = bool(features & Clusters.TimeSynchronization.Bitmaps.Feature.kNTPClient)
+        self.supports_ntps = bool(features & Clusters.TimeSynchronization.Bitmaps.Feature.kNTPServer)
+        self.supports_trusted_time_source = bool(features & Clusters.TimeSynchronization.Bitmaps.Feature.kTimeSyncClient)
+
+        time_cluster = Clusters.Objects.TimeSynchronization
```

Comment:
```suggestion
        time_cluster = Clusters.TimeSynchronization
```

Reply:
This isn't wrong, but we moved these back into the Clusters section in the init (no sure why...whatever), but if we keep this consistent, it's easier for folks to know what's going on